roles/news/templates/var_www_selfoss_config.ini.j2

```diff
@@ -1,9 +1,10 @@
 ; see https://github.com/SSilence/selfoss/wiki/Configuration for more information about the configuration parameters
 [globals]
 db_type=pgsql
-db_database={{selfoss_db_database}}
-db_username={{selfoss_db_username}}
-db_password={{selfoss_db_password}}
+db_host=127.0.0.1
+db_database="{{selfoss_db_database}}"
```

Comment:
I'm not sure what the escape character are, commas certainly. Prevents against escape characters causing issues in the username as well, in case a user randomly generates their username.